build/custom.mk

```diff
@@ -0,0 +1,5 @@
+# Include custom targets and environment variables here
+ifndef MM_RUDDER_WRITE_KEY
+	MM_RUDDER_WRITE_KEY = 1d5bMvdrfWClLxgK1FvV3s4U1tg
+endif
+GO_BUILD_FLAGS += -ldflags '-X "github.com/mattermost/mattermost-plugin-jira/server/utils/telemetry.rudderWriteKey=$(MM_RUDDER_WRITE_KEY)"'
```

Comment:
The Makefile in this repo is not importing this `custom.mk` file. You'll need to import it like other repos are using it.

Reply:
@mickmister is right. This is at the moment not working. Probably the reason he stated.
One way to check it is to see the `make build` commands used. Checking the CI, I can see the following:

`/usr/local/go/bin/go build  -ldflags '-X "main.BuildDate=Thu Aug 13 15:45:30 UTC 2020" -X "main.BuildHash=7003f3fe9d814767f77a83f58b9f36101c877dec" -X "main.BuildHashShort=7003f3f"' -o dist/plugin-darwin-amd64;`

There you should have also the other ldflag in place.

On top of that, if the environment variable is set on CircleCI, you will see the environment variable censored, instead of raw. If you see the development variable raw on CircleCI it means that you have not set the variable correctly on CircleCI.

As a side note, setting the variable in CircleCI is not by itself part of this PR (it is done directly on circleCI and it is not linked to any specific PR) but it would be great to have it done now, so we don't forget.

Reply:
Thanks, fellas. I added the import of `custom.mk` to the makefile.  I tested locally to ensure the variable was accessible in the telemetry package and defaulted to the dev value is `MM_RUDDER_WRITE_KEY` was not set.  I also tested the value matched `MM_RUDDER_WRITE_KEY` if it was set.

Reply:
I also added the Environment variable to CircleCI. Here is the redacted result.
![image](https://user-images.githubusercontent.com/7575921/90278389-0232ce00-de2d-11ea-9391-9546f2fd85d5.png)